common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -38,10 +97,876 @@ innovation_taming_the_skies = {
 
 	potential = {
 		is_giant_culture_trigger = no
+		NOT = {
+			has_aerial_maa_trigger = yes
+		}
 	}
 
 	unlock_maa = aerial_unit
 
 	flag = global_regional
 	flag = tribal_era_regional
 }
+
+### MAA TECHS
+
+####Tribal
+innovation_archmage = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @archmage
+
+	potential = {
+		AND = {
+			OR = {
+				this.culture_group = culture_group:human_group
+				this = culture:blood_elf
+				this = culture:high_elf
+			}
+			NOT = {
+				this = culture:pirate_culture
+				this = culture:wastewander	
+			}
+		}
+	}
+
+	unlock_maa = archmage
+
+	flag = global_maa
+}
+
+
+innovation_blademaster = {
```

Comment:
Not the best art.
First, that's not how blademasters look, their look is very unique.
Second, you see a blood elf.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/133155771-ab19f08f-18d5-421d-bd25-23e63bb7d5b8.png)

</details>

The icon isn't great too since there's blademaster unit in Reforged. You can take his icon.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/133155861-53cda9bf-8114-433c-a621-89654c53d36a.png)

</details>

Reply:
👌